config/setup_pre_4.4.bash

```diff
@@ -24,7 +28,7 @@ chmod -R 777 ../content/collicon
 chmod -R 777 ../content/dwca
 chmod -R 777 ../content/geolocate
 chmod -R 777 ../content/imglib
-chmod -R 777 ../content/lang
```

Comment:
Shouldn't we restrict the write access to the group here since php will be the only thing writing to it? Or does that just end up being a hassle?

Reply:
It's a good question.  Based on the principle of least privilege you are right.  But it does cause some additional hassle.